number_theory/modular_unittest.cpp

```diff
@@ -0,0 +1,41 @@
+#include <gtest/gtest.h>
+
+#include "number_theory/modular.h"
+
+namespace tql {
+
+TEST(ModularTest, Basic) {
+  using Mod10 = Modular<int, 10>;
+
+  // test constructors
+  Mod10 a = 123;
+  EXPECT_EQ(a.get(), 3);
+  Mod10 b = a;
+  EXPECT_EQ(b.get(), 3);
+  Mod10 c;
+
+  // test assignments
```

Comment:
I think reusing same variables make it harder to keep track of their values and therefore it makes the tests harder to read. I suggest using either different variables or explicitly re-initializating them at the start of each test block.